README.md

```diff
@@ -94,13 +104,11 @@ for i, key := range keys {
 - **Key Distribution**: Automatically routes keys to the correct node based on hash
```

Comment:
With respect to automatic retries, it would be nice to include some more insights, e.g. for how long using what strategies, use cases for partial failures, etc.

Reply:
Will add more info :+1:

Reply:
@atzoum please review https://github.com/rudderlabs/keydb/pull/41/commits/f0be17a59a637fd4f69e0aafc8ed9cb6168a68c4

Reply:
@atzoum added more docs around `/autoScale` https://github.com/rudderlabs/keydb/pull/41/commits/a00c33cf4dcaa39a17e93a335b64346176f6400f